src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -12,10 +12,52 @@ public class HelpCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String SHOWING_HELP_MESSAGE = "Available Commands:\n"
+            + "- help: Shows details on available commands\n"
+            + "- add: Adds a new contact\n"
+            + "- delete: Removes a contact\n"
+            + "- list: Displays stored contacts";
+
+    private final String ShowingCommandMessage;
```

Comment:
```suggestion
    private final String showingCommandMessage;
```
Should member variables follow camelCase naming format?